documentation/docs/getting_started/nodes_101.md

```diff
@@ -17,35 +17,35 @@ called nodes, and nodes are the backbone of an IOTA network. In this article, yo
 Nodes are responsible for:
 
 - Providing an API to interact with the Tangle/IOTA network.
-- Validating [messages](https://chrysalis.docs.iota.org/guides/dev_guide#messages-payloads-and-transactions) and ledger mutations for consistency.
+- Validating [blocks](https://chrysalis.docs.iota.org/guides/dev_guide#blocks-payloads-and-transactions) and ledger mutations for consistency.
```

Comment:
Currently wrong, will maybe be updated to this?

Reply:
Nah, it's chrysalis

Reply:
I'll keep it like that for now. We need to redo the doc anyway.